sdk/ai/Azure.AI.Agents/samples/Sample8_FileSearch.md

```diff
@@ -0,0 +1,160 @@
+# Sample file search with agent in Azure.AI.Agents.
+
+In this example we will create the local file, upload it to Azure and will use it in the newly created `VectorStore` for file search.
+
+1. First, we need to create agent client and read the environment variables, which will be used in the next steps.
+
+```C# Snippet:Sample_CreateAgentClient_FileSearch
+var projectEndpoint = System.Environment.GetEnvironmentVariable("PROJECT_ENDPOINT");
+var modelDeploymentName = System.Environment.GetEnvironmentVariable("MODEL_DEPLOYMENT_NAME");
+AgentsClient client = new(endpoint: new Uri(projectEndpoint), tokenProvider: new DefaultAzureCredential());
+OpenAIClient openAIClient = client.GetOpenAIClient();
+```
+
+2. We will create a toy example file and upload it using OpenAI mechanism.
+
+Synchronous sample: 
+```C# Snippet:Sample_UploadFile_FileSearch_Sync
+string filePath = "sample_file_for_upload.txt";
+System.IO.File.WriteAllText(
```

Comment:
```suggestion
File.WriteAllText(
```
I believe it's OK for us to omit most direct namespace use in samples, particularly common ones under `System`.

Reply:
Fixed.